lib/tests/modules.sh

```diff
@@ -82,6 +86,24 @@ checkConfigOutput() {
     fi
 }
 
+checkJsonOutput() {
+  local path=$1
+  local actual expected diff
+  actual="$(local-nix-instantiate -A actual "$path")"
+  expected="$(local-nix-instantiate -A expected "$path")"
+  diff=$(jd <(echo "$actual") <(echo "$expected") || true)
```

Comment:
This introduces an additional dependency `jd-diff-patch` I am not sure how we want to go about this.
Previously i could just run `modules.sh`, with this change i need to load the tool into my shell first.
I don't like doing this manually.

Reply:
Okay, diffs are nicer to work with but applied.